next/components/forms/simple-components/Accordion.tsx

```diff
@@ -90,7 +99,36 @@ const Accordion = ({
                 'text-p-md': size === 'lg' || size === 'md',
               })}
             >
-              {children}
+              <ReactMarkdown
+                className={cx('flex flex-col gap-4', className)}
+                remarkPlugins={[remarkGfm, remarkDirective, remarkDirectiveRehype]}
+                rehypePlugins={[rehypeRaw, remarkDirective, remarkDirectiveRehype]}
+                components={
+                  {
+                    h1: ({ children }: any) => <h1 className="text-h1 font-bold">{children}</h1>,
+                    h2: ({ children }: any) => <h2 className="text-h2 font-bold">{children}</h2>,
```

Comment:
```suggestion
```

we disable h1 and h2 on purpose - headers of 'this level' should not be contained within the accordion